src/main/java/racingcar/service/RacingCarService.java

```diff
@@ -0,0 +1,67 @@
+package racingcar.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import racingcar.model.Car;
+import racingcar.utlis.Convertor;
+import racingcar.validator.NameValidator;
+
+public class RacingCarService {
+
+	private List<Car> cars;
+
+	public RacingCarService(List<Car> cars) {
+		this.cars = cars;
+	}
+
+	public void playRound() {
+		cars.forEach(Car::decideMove);
+	}
+
+	public void saveCars(String inputCarNames) {
+		List<String> carNames = Arrays.asList(Convertor.separateNamesByDelimiter(inputCarNames));
+		NameValidator.checkNameList(carNames);
+		save(carNames);
+	}
+
+	private void save(List<String> carNames) {
+		for (String name : carNames) {
+			cars.add(new Car(name, 0));
+		}
+	}
+
+	public int findMaxPosition() {
+		int maxPosition = 0;
+		for (Car car : cars) {
+			maxPosition = getMaxPosition(maxPosition, car);
+		}
+		return maxPosition;
+	}
+
+	private int getMaxPosition(int maxPosition, Car car) {
+		if (car.isOverMaxPosition(maxPosition)) {
+			maxPosition = car.getPosition();
+		}
+		return maxPosition;
+	}
```

Comment:
stream api를 이용하면 아래처럼 간단히 구현도 가능합니다.
```suggestion
	private int findMaxPosition() {
	        return cars.stream()
	                .mapToInt(Car::getPosition)
	                .max()
	                .orElse(0);
	}
```